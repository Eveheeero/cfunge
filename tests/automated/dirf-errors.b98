"FRID"4(v ; Test error handling for DIRF ;
        >     n #vC na"C :DAB"  >:#,_v
v                >  na"C :DOOG" >:#,_>

>             n #vM na")1( M :DAB"  >:#,_v
v                >  na")1( M :DOOG" >:#,_>

>         "b/a" #vM na")2( M :DAB"  >:#,_v
v                >  na")2( M :DOOG" >:#,_>

>             n #vR na")1( R :DAB"  >:#,_v
v                >  na")1( R :DOOG" >:#,_>

>      "hcuson" #vR na")2( R :DAB"  >:#,_v
v                >  na")2( R :DOOG" >:#,_>
@