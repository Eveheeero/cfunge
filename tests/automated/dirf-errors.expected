GOOD: C
GOOD: M (1)
GOOD: M (2)
GOOD: R (1)
GOOD: R (2)
